templates/github/.github/workflows/publish.yml.j2

```diff
@@ -66,27 +66,45 @@ jobs:
       - name: "Upload python client packages"
         uses: "actions/upload-artifact@v4"
         with:
-          name: "python-client.tar"
+          name: "{{ plugin.app_label }}-python-client.tar"
           path: "{{ plugin_name }}/{{ plugin.app_label }}-python-client.tar"
           if-no-files-found: "error"
           overwrite: true
 
+      - name: "Merge Python client packages"
```

Comment:
Can we use this instead?

https://github.com/actions/upload-artifact?tab=readme-ov-file#upload-using-multiple-paths-and-exclusions

Reply:
Sure.